src/NoteDataUtil.cpp

```diff
@@ -961,11 +961,47 @@ struct recent_note_t
 		:row(r), track(t) {}
 };
 
+// CalculateRadarValues has to delay some stuff until a row ends, but can
+// only detect a row ending when it hits the next note.  There isn't a note
+// after the last row, so it also has to do the delayed stuff after exiting
+// its loop.  So this state structure exists to be passed to a function that
+// can be called from both places to do the work.  If this were Lua,
+// DoRowEndRadarCalc would be a nested function. -Kyz
+struct crv_state_t
```

Comment:
Uh, there's a 6 line comment explaining exactly what it's for to make everything clear.  I could name it with the full name of the function if using the initials is really a problem.
Yes, the underscore style is going to stay because camel case is less readable.  And the _t is going to stay because it's the name of a type.